pytorch_lightning/loggers/logger.py

```diff
@@ -346,6 +346,17 @@ def __iter__(self):
         # if DummyLogger is substituting a logger collection, pretend it is empty
         yield from ()
 
+    def __getattr__(self, name: str) -> Callable:
```

Comment:
Note, now that is is implemented, getattr would be called on any attribute lookup that cant be relsolved by getattribute(). We assume the attribute is always a method, which is the best we can do I guess.